### ISSUE TITLE
Migrations is missing

### ISSUE THREAD

**Issue by mikaelengstrom** [mingchen/django-cas-ng#75]
Hi, this lib breaks my test suit, it reports:
```
django.db.utils.ProgrammingError: relation "auth_user" does not exist"
```
If i run `python manage.py makemigrations django_cas_ng && python manage.py migrate` it works again.

pip freeze:
```
boto==2.38.0
click==6.0
Django==1.8.7
django-cas-ng==3.5.3
django-debug-toolbar==1.3.0
django-filter==0.11.0
django-mama-cas==1.2.0
django-mptt==0.7.4
django-nose==1.4.2
django-reversion==1.9.3
django-rosetta==0.7.6
django-storages-redux==1.3
django-suit==0.2.15
django-wysiwyg-redactor==0.4.9
djangorestframework==3.3.1
djangorestframework-gis==0.9.6
ecdsa==0.13
Fabric==1.10.0
geopy==1.11.0
gitdb==0.6.4
GitPython==1.0.1
Jinja2==2.8
MarkupSafe==0.23
microsofttranslator==0.5
nose==1.3.7
paramiko==1.16.0
pipdeptree==0.4.3
polib==1.0.7
psycopg2==2.6.1
pycrypto==2.6.1
python-cas==1.1.0
python-dotenv==0.1.3
requests==2.8.1
six==1.10.0
smmap==0.9.0
sqlparse==0.1.18
Unipath==1.0
wheel==0.24.0
```

**Comment by nitmir** [comment]
I try build a migration history soon

**Comment by nitmir** [comment]
did you ran ```python manage.py syncdb``` ?

**Comment by mikaelengstrom** [comment]
Yep, I have tried that, but i need to run makemigrations first, otherwise it does not work.

**Comment by nitmir** [comment]
Migration are new in django 1.7 but django-cas-ng is suppose to work with django 1.5 and 1.6, thus I do not think migrations will be added to the repository.
Normally, django create the models tables for the apps without migrations as it did in version previous 1.7.

I think that you experienced is describe here https://docs.djangoproject.com/en/1.8/topics/migrations/#dependencies : id est, django-cas-ng model have a foreign key to user auth model. django-cas-ng is an unmigrated but the default auth model is in a migrated app and thus database creation are not happening in the right order.

The solution may be to run ```python manage.py migrate``` a first time before added django-cas-ng to ``INSTALLED_APPS``` and then add django-cas-ng to ``INSTALLED_APPS``` and run again ```python manage.py migrate```.

The foreign key is here to allow cascade deletion: If a user is deleted all his ProxyGrantingTicket are deleted. As ProxyGrantingTicket allow django to request ProxyTicket (cas be seen as OTP passwords) for the user, it has seemed me safer to do it like this.
As the django doc says it is a bad idea, I may send a pull request deleting this foreign key and do ProxyGrantingTicket cleaning in a management command.

Can you confirm that the problem you encounter can be worked around by doing a two step migration ?

**Comment by mikaelengstrom** [comment]
I did not have any problems to migrate django-cas-ng when first installing it (dependencies were already migrated). It is when `./manage.py test` is creating the test-database the migration break.

So yes, i can confirm that a two step migration is working and that the assumptions in your comment is correct.

But i don´t understand why we can´t just add migrations to the app? Would it break backward compatibility?

**Comment by kavdev** [comment]
@mikaelengstrom: It wouldn't. You would just have migrations work for Django 1.7+. 1.8 is now LTS, so it shouldn't be a problem.

**Comment by kavdev** [comment]
until that happens, just ```manage.py makemigrations django_cas_ng``` and then
```manage.py migrate```

**Comment by mingchen** [comment]
Fixed by #78 with change 4c4cf87a.
